src/DataTable/tests/FilterStatus.test.jsx

```diff
@@ -61,8 +55,4 @@ describe('<FilterStatus />', () => {
     const wrapper = mount(<FilterStatusWrapper value={{}} props={filterProps} />);
     expect(wrapper.text()).toEqual('');
   });
-  it('hides filter text', () => {
-    const wrapper = mount(<FilterStatusWrapper value={instance} props={filterPropsNoFiltered} />);
-    expect(wrapper.text()).toEqual('');
-  });
```

Comment:
Should we be removing this test? Is there another test covering the `showFilteredFields={false}` case? There is at least one or two teams using this prop to hide the "Filtered by..." text because it shows the `accessor`'s value which may be a programmatic value, e.g. "Filtered by requestStatus".

Reply:
Yeah, my bad...fixed it and reverted the change, glad you notice, thanks!